axfr.go

```diff
@@ -14,6 +14,34 @@ func (g *Gandi) ListTsigs() (tsigs []Tsig, err error) {
 	return
 }
 
+// By gandi API design, an axfr/tsig/uuid/config/[bind|knot|powerdns|nsd] request will not return a valid JSON,
+// but rather the actual configuration example.
+// Hence we have to treat the return as 'configexample string' instead of 'configexample []string'.
+
+// ListTsigBind shows a BIND nameserver config, and includes the nameservers available for zone transfers
+func (g *Gandi) ListTsigBind(uuid string) (configexample string, err error) {
```

Comment:
Please name the variable `configExample` instead of `configexample`, let's keep the code as beautiful as possible.